autogtp/README.md

```diff
@@ -4,21 +4,45 @@ This is a self-play tool for Leela-Zero. When launched, it will fetch the
 best network from the server so far, play a game against itself, and upload
 the SGF and training data at the end of the game.
 
+To work successfully, autogtp binary needs to also have leelaz binary to be in the same directory.
```

Comment:
Shorten and make clearer: AutoGTP expects to find the leelaz binary in the same directory.

Reply:
your sentence is shorter, but i dont find it clearer

i'll try something else shorter, for you to review it again

Reply:
tried another shorter sentence